Makefile

```diff
@@ -0,0 +1,34 @@
+## This file is part of the adac-hybrid-arm distribution
+## Copyright (C) 2020, European Space Agency
+##
+## This program is free software: you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation, version 3.
+##
+## This program is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+SRC_DIR=src
+
+.PHONY: all test adac rt clean
+
+all: adac rt test
+
+adac:
+	( ${MAKE} -C ${SRC_DIR}/arm-elf-adac) || exit $$?;
```

Comment:
Is this `|| exit $$?;` required? If children make fails then the parent fails too

Reply:
Yes, that is the desired outcome